components/Navigation/navigation.css

```diff
@@ -25,3 +25,51 @@
 .nav-icon-btn {
   color: var(--primary-color);
 }
+
+/* navbar */
+.brand-name {
+  color: var(--primary-color);
+}
+.text-dec-none {
+  text-decoration: none;
+}
+.search-box {
+  width: 50rem;
```

Comment:
i think its better to have 1 line space between CSS blocks for better readability

Reply:
ok